test/options_test.cc

```diff
@@ -249,7 +250,7 @@ TEST_F(OptionsImplTest, AlmostAll) {
   EXPECT_TRUE(util(cmd->stats_sinks(0), options->statsSinks()[0]));
   EXPECT_TRUE(util(cmd->stats_sinks(1), options->statsSinks()[1]));
   EXPECT_EQ(cmd->latency_response_header_name().value(), options->responseHeaderWithLatencyInput());
-
+  // TODO(#433)
```

Comment:
Two nit asks on this set of TODOs:
1.  Can we add a oneliner, explaining what the action to do here specifically is
2.  Rather than repeating it on every test here, we could have in the comment something like "Here and below".  Then we only need to include the TODO once per relevant file

Reply:
1. sounds good to me. I feel like tagging the comparisons which need to be replaced is helpful, but I can remove the other ones if you insist.